server.py

```diff
@@ -0,0 +1,188 @@
+import pickle
+import socket
+import sys
+import threading
+
+import numpy as np
+
+SERVER_QUIT = [None, None, None]
+CONNECTION_IS_DEACTIVATED = False
+CONNECTION_IS_ACTIVE = True
+BACK_TO_CHAMPION_SELECT = "B"
+CHOSEN_CHAMPIONS_INFORMATION_REQUEST = "P"
+CHAMPION_IS_AVAILABLE = b"YES"
+CHAMPION_IS_NOT_AVAILABLE = b"NO"
+QUIT_SIGNAL = b"Q"
+CLIENT_QUIT = -1
+
+CHAMPION_IS_NOT_CHOSEN = -1
+NEW_CONNECTION = 1
+
+CONNECTION_IS_SUCCESSFUL = b"OK"
+SERVER_IS_FULL = b"NO"
+
+SERVER_IS_CLOSING = True
+SERVER_IS_RUNNING = False
+
+CLOSED_CONNECTION = 1
+QUIT = "q"
+
+GAME_IS_ENDED = 1
+MAX_PLAYERS = 3
+NEW_GAME = 1
+
+SERVER_IP = "127.0.0.1"
+SERVER_PORT = 6001
+BUFFER_SIZE = 4096
+
+NUMBER_OF_THREADS_TO_CANCEL_CONNECTION = 1
+NUMBER_OF_STARTED_CONNECTIONS_INIT_VALUE = 0
+NUMBER_OF_STARTED_GAMES_INIT_VALUE = 0
+NOT_NEED_OF_NEW_CONNECTIONS = 0
```

Comment:
You Could group this constants like
--Server consts--
.
.
.
--Thread consts--
.
.
.
--Connections-- 
etc.

Reply:
Yeah, you right. Grouped and commented